about.html

```diff
@@ -0,0 +1,311 @@
+                                <br> launches to support diversity
+                            </h1>
+
+                        </div>
+
+                    </div>
+
+
+
+                    <div class="row">
+                        <div class="col-12">
+                            <p class="about_paraghraph">
+                                <span class="about_span">African Women in Tech is a term that refers to the community of women </span> African descent
+                                who work in the technology industry. This includes women who work in fields such as software
+                                development, data science, computer engineering, information technology, and other related
+                                fields.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="row">
+
+                        <div class="col-12 about_paragraph_second">
+
+                            <p>
+                                <p>African women have traditionally </p> underrepresented in the technology industry, and the
+                                goal of African Women in Tech groups is to help increase their representation and provide
+                                support for those who are already working in the field. This can include providing access
+                                to resources, mentorship, and networking opportunities. There are a number of initiatives
+                                and organizations that focus on African Women in Tech, with varying mission and focus. Some
+                                focus on providing training and education to women to help them acquire the skills needed
+                                to enter the technology industry, while others focus on providing networking and mentorship
+                                opportunities to help women advance in their careers.
+
+                            </p>
+                            <p>
+                                <img src="images/about_icon.png" alt="icon">
+                            </p>
+                            <p class="about_paragraph_box">October 14th - 17th, 2015 (Wed - Sat)
+                                <br> See you at African Women in Tech Summit!</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-12 ">
+                            <p class="about_buttom_paragraph">Other questions about African Women in Tech?
+                                <br>
+                                <span class="organisation">
+                                    <a href="#">AfricanWomenin Tech.org</a>
+                                </span>
+                            </p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </div>
+        <div>
```

Comment:
- [ Required ] Please consider changing this tag to the HTML5 semantic `section` tag, the `section` tag is usually used to wrap logical sections of a page that have `hx` tags, since this is a logical section of your page please wrap it in a `section` tag as shown below. 
```HTML
<section class="container">
  <div class="row">
     <div class="col-1 desk-div"></div>
       <div class="col-11">
           <h1 class="main_header">African women in tech group <br> launches to support diversity</h1>
        </div>
   </div>
</section>
```